Scrap_Dbpedia_Composers/dbPedia_18thCentComposers.js

```diff
@@ -0,0 +1,182 @@
+const http = require('http');
```

Comment:
Why is it necessary to have separate files for each century? Does the structure differ too much from each other?